scripts/postinstall.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+/* eslint-disable */
+/* Adapted from nodemon's postinstall: https://github.com/remy/nodemon/blob/master/package.json */
+
+var msg = 'Take a look at https://opencollective.com/styled-components to help out styled-components’ maintainers';
```

Comment:
"Use styled-components at work? 💅 Consider supporting our development efforts at https://opencollective.com/styled-components"

Alternatively "Love styled-components?"

Reply:
I like first one :) that’s more specific I guess 👌

Reply:
updated the wording to the first option :) removed the emoji — as described in the PR message — I don't think it'd fair well with some